### ISSUE TITLE
Make an event and delivery proposal for team + CoCo capabilities strategy

### ISSUE THREAD

**Issue by MarianaVazquezCYF** [CodeYourFuture/CYF-Staff-Tickets#666]
At Code Your Future, we aim to create an event page where individuals can explore and join our diverse range of signature events, leveraging the power of platforms like Meetup effectively. These are some examples: We are developers events page with their siganute events: https://www.wearedevelopers.com/live and coding black females meet up page: https://www.meetup.com/coding-black-females/events/

The signature events I would like CYF to have will be:

1. Festival of Diverse Talent
 2. Ladies Brunch in Tech
3. Tech Confessions
4. Portfolio Saturdays
5. Coding Challenges
6. Throwback Demos Fridays
7. Summit (internal people) or Conference (external people) 

In the upcoming proposal, I will outline the objectives and delivery strategies for each of these events. A crucial aspect of this plan is addressing the challenge of finding volunteers willing to take the lead in organizing and executing these events. To tackle this, I propose:

1. Launching a targeted campaign to recruit community coordinator volunteers, clearly specifying the roles and responsibilities involved in delivering these events. For example in the past, we recruited community coordinators to do random things or for them to figure out what to do, however this require a lever of leadership that no a lot of our cocos have! that's why I want to recruit volunteers were we specify their role and their project. 
2. Creating an event roadmap on GitHub, detailing the step-by-step process and best practices for organizing successful events, empowering volunteers with the necessary resources.

Currently, the primary struggle lies in volunteers expressing interest only for one-off contributions or trainees expressing personal challenges, which has prolonged the implementation of this strategy. Sometimes I want to to undertake these events by myself with minimal community support, however I know my role as the head of the community should be to foster an environment where everyone feels empowered to take the lead and contribute to the creation of engaging events and initiatives.

I recognize the frustration that comes with this challenge, but I remain committed to overcoming it. By developing a detailed proposal and seeking your feedback, I aim to refine our approach and move forward with this new plan, ultimately building a vibrant and collaborative community where members actively shape and drive our events.! 

stakeholders and approvers: 
@lilianabermudes @alecmccrindle @MarianaVazquezCYF 

I will share the proposal on this ticket when I have it ready !

**Comment by alecmccrindle** [comment]
Thanks for sharing the idea of sharing a detailed proposal. Having a way of running events will be great. 

Here is my feedback:

1. I think the event page is a good thing to work on, but probably a separate ticket?
2. The Rubric mentions the following 'events to bring the whole community together, such as the summer and winter socials.','The region has team members who organise a monthly meet-up to get to know CYF with external people',' There is a CYF summit twice a year in different regions.','monthly meetings with the partnerships-sponsors'. We also have the CYF conference, which I understand is essential. Will the proposal cover these other meetings? Do we think a new volunteer team will be able to do all of them at once, and if not, how can we decide which priorities to make?  
3. You mention the current challenges that volunteers face at the moment. In the proposal, will you share a list of all the fundamental problems that volunteers have faced in organising events? Would speaking to current and former CoCo volunteers about the challenges might be useful? I think @lilianabermudes and I can help you with our experiences, too.

**Comment by MarianaVazquezCYF** [comment]
- 1. The event page is part of the proposal, no worries once the proposal is approved I can make all the tickets to make it possible
- 2. summer and winter socials are events just for the community that anyone can organise and these events are to bring the community together, not to bring external people (although they can of course) ! the events that I mentioned before + the cyf conference ), are events that are more branded and more directed to bring external people, volunteers or tech leaders! 
- 3. Perfect, I will share a list and I will ask the community as well what are the challenges they have seen! 
- 4. ps: I will add the conference to this part of events! conference - something like uncodebar right? were we bring the community together, not the summit which is just for the community?

**Comment by lilianabermudes** [comment]
Initial thoughts:

1. The events page is something we definitely need :-)
2. When you say target campaign you mean external campaign or internal comms to recruit existing community members?
3. I believe the training with Roberta will also help you, Mariana to identify how you will be able to guide the community --> what you, and "only" you, can do as part of your role (the owner of the events) x what the community can do

**Comment by kfklein15** [comment]
@MarianaVazquezCYF , when you look at this, please create 3 milestones and add 1 placeholder ticket to each.
1. Events page (ticket 1: research options of event software available) 
2. Format of community-led events (ticket 1: research)
3. Run the events (each event should have each own milestone or ticket) (i.e. Job fair in another location)

**Comment by MarianaVazquezCYF** [comment]
Proposal ready:
https://docs.google.com/document/d/1Xmta7Zj8jhQyIR1rfggUx5gWb0kHKEpPrenxdihn-YY/edit. The roadmaps are not ready yet, but I made a ticket for those
